__tests__/utils/helpers.js

```diff
@@ -24,3 +27,26 @@ export const createUser = user =>
         email: user.email,
         password: 'secret0001'
     });
+
+export const createAdmin = user =>
+    User.query().insert({
```

Comment:
I think instead of having a method called `createAdmin`, let's instead pass an object to the `createUser` method to customise it. For example it could look like this:

```js
export const createUser = user => User.query().insert({
   role: user.role || 'patron'
})
```
That way we can use the same method for both admin, patron and super admin